ee/log/osquerylogs/log.go

```diff
@@ -96,21 +90,23 @@ func (l *OsqueryLogAdapter) Write(p []byte) (int, error) {
 func (l *OsqueryLogAdapter) logInfoAboutUnrecognizedProcessLockingPidfile(p []byte) {
 	matches := pidRegex.FindAllStringSubmatch(string(p), -1)
 	if len(matches) < 1 || len(matches[0]) < 2 {
-		level.Debug(l.logger).Log(
-			"msg", "could not extract PID of non-osqueryd process using pidfile from log line",
+		l.slogger.Log(context.TODO(), slog.LevelDebug,
```

Comment:
Oh, debug logs don't get shipped except during startup, right? If that's correct, would you please update all of the logs in `logInfoAboutUnrecognizedProcessLockingPidfile` and the functions it calls to be info or error level? We haven't been able to reproduce that error at all, and it would be super cool if we can ship it to the cloud logs and find it!

Reply:
updated to log an before we start dumping info about locking process (the info dump is info level or error if we encounter error getting info)